pkg/provider/azure_storageaccount.go

```diff
@@ -405,7 +425,7 @@ func (az *Cloud) createPrivateEndpoint(ctx context.Context, accountName string,
 	privateLinkServiceConnection := network.PrivateLinkServiceConnection{
 		Name: &privateLinkServiceConnectionName,
 		PrivateLinkServiceConnectionProperties: &network.PrivateLinkServiceConnectionProperties{
-			GroupIds:             &[]string{GroupIDFile},
```

Comment:
what about setting `GroupIds` as both `file`, `blob`, by that way, this PrivateLinkServiceConnection could be used for both  blob and file? that may be more straightforward